netbox/utilities/templatetags/form_helpers.py

```diff
@@ -120,13 +127,48 @@ def render_field(field, bulk_nullable=False, label=None):
     """
     Render a single form field from template
     """
+
     return {
         'field': field,
         'label': label or field.label,
         'bulk_nullable': bulk_nullable or getattr(field, '_nullable', False),
     }
 
 
+@register.inclusion_tag('form_helpers/render_field.html')
+def render_table_filter_field(field, table, request):
+    """
+    Render a single form field for table column filters from template
+    """
+    url = ""
+
+    # Handle filter forms
+    if table:
+        # Build kwargs for querystring function
+        kwargs = {field.name: None}
+        # Build request url
+        if request and table.htmx_url:
+            url = table.htmx_url + querystring(request, **kwargs)
+        elif request:
+            url = querystring(request, **kwargs)
+        # Set HTMX args
```

Comment:
minor - what code is this comment referring to?  If the code below can it be indented with it.